Assets/HoloToolkit-Examples/SpatialSound/Scenes/AudioLoFiOcclusionTest.unity

```diff
@@ -0,0 +1,686 @@
+%YAML 1.1
```

Comment:
This demo scene needs updating to the new standards defined by @cre8ivepark 
Should have:
* Information dialog describing scene
* More information as to what it is demonstrating
* Should be more of a Show normal, LoFi and HiFi so as to better demonstrate what the scripts provide. (use gaze to activate /  deactivate each)

Reply:
in progress

Reply:
Example scene updated and renamed. Removed the occlude as that is not what is being demonstrated.

Scene showcases three of the five sound quality settings.